doc/Known_issues/CredSSP_error_bestiary.md

```diff
@@ -0,0 +1,28 @@
+# I get authentication error "CredSSP encryption Oracle remediation" when trying to RDP into a Windows machine from another Windows machine
+
+![CredSSP encryption Oracle remediation](CredSSPError.png)
+
+
+If you see the above error, please try the following two options:
```

Comment:
Reading through the options, it's not clear to me whether I need to do one of them, or both of them? 

If it's the former, then

```suggestion
If you see the above error, please configure both the client and the server:
```

If it's the latter, then 

```suggestion
If you see the above error, please configure either the client or the server:
```

Reply:
ok